src/main/resources/init_db.sql

```diff
@@ -9,6 +9,8 @@ CREATE TABLE `drivers`  (
                             `id` bigint(0) UNSIGNED NOT NULL AUTO_INCREMENT,
                             `name` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
                             `license_number` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                            `login` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
+                            `password` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
```

Comment:
Your login should be unique. We cant create users with same login

Reply:
done